lms/templates/courseware/course_navigation.html

```diff
@@ -83,27 +78,14 @@
 % if disable_tabs is UNDEFINED or not disable_tabs:
 <nav class="${active_page} wrapper-course-material" aria-label="${_('Course Material')}">
   <div class="course-material">
-    <ol class="course-tabs">
-      % for tab in get_course_tab_list(request, course):
-        <%
-            tab_is_active = (tab.tab_id == active_page) or (tab.tab_id == default_tab)
-        %>
-          <li>
-             <a href="${tab.link_func(course, reverse) | h}" class="${url_class(tab_is_active)}">
-                 ${_(tab.name) | h}
-                 % if tab_is_active:
-                   <span class="sr">, current location</span>
-                 %endif
-                 % if tab_image:
-                   ## Translators: 'needs attention' is an alternative string for the
-                   ## notification image that indicates the tab "needs attention".
-                   <img src="${tab_image}" alt="${_('needs attention')}" />
-                 %endif
-             </a>
-          </li>
-      % endfor
-      <%block name="extratabs" />
-    </ol>
+     <%
+      base_tabs_tpl = 'courseware/tabs.html'
+      tabs_tmpl = static.get_template_path(base_tabs_tpl)
+      if tabs_tmpl == base_tabs_tpl:
+          tabs_tmpl = '/' + base_tabs_tpl
```

Comment:
@StraNNiKK I have issued PR #11548 to fix absolute path issue with `get_template_path` so you should be able to use it without putting that clutter logic of prepending `/. You should be able to use it like this in both microsite and non microsite cases.
`<%include file="${static.get_template_path('/courses/tabs.html')}" args="tab_list=tab_list,active_page=active_page,default_tab=default_tab,tab_image=tab_image" />
`